test/FunctionalTests/Microsoft.OData.Core.Tests/Build.NetFramework/Microsoft.OData.Core.Tests.csproj

```diff
@@ -437,9 +441,6 @@
       <Name>Microsoft.Test.OData.Utils</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup>

```

Comment:
This should not be removed. I think this was mismatched during one of my rebases. Thanks for the catch!